webapp/packages/core-ui/src/Tabs/TabsState.tsx

```diff
@@ -151,7 +163,7 @@ export const TabsState = observer(function TabsState<T = Record<string, any>>({
       tabId: currentSelectedId,
       props,
     });
-  }, [currentSelectedId]);
+  }, [currentSelectedId, dynamic.selectedId, openExecutor, props]);
```

Comment:
This might change behaviour unexpectedly. Probably the purpose of this effect is only to fire `openExecutor` when a new tab is selected, but now it will be fired in many other cases. I recommend not changing this until you encounter a problem with the current behavior.